keymaps/git.cson

```diff
@@ -3,6 +3,19 @@
   'ctrl-shift-8': 'github:toggle-github-tab'
   'ctrl-9': 'github:toggle-git-tab-focus'
   'ctrl-shift-9': 'github:toggle-git-tab'
+  'alt-g enter': 'github:commit'
+  'alt-g f': 'github:fetch'
+  'alt-g F': 'github:pull'
```

Comment:
@nathansobo It looks like it was updated so that we should always see the `alt-shift-S` form post-normalization:

https://github.com/atom/atom-keymap/blob/master/spec/helpers-spec.coffee#L8

From the specs it appears that any sub-sequence with a `shift` and an alphabetic character will capitalize the alphabetic character in the normalized form. Personally, I don't think that's a big deal ... but it would probably be less confusing if we normalized to all lower case.